test/cloudtest/test_upgrade.py

```diff
@@ -42,10 +42,17 @@ def actions(self) -> List[Action]:
 
 
 @pytest.mark.long
-def test_upgrade(aws_region: Optional[str]) -> None:
+def test_upgrade(
+    aws_region: Optional[str], log_filter: Optional[str], dev: bool
+) -> None:
     """Test upgrade from the last released verison to the current source by running all the Platform Checks"""
 
-    mz = MaterializeApplication(tag=str(LAST_RELEASED_VERSION), aws_region=aws_region)
+    mz = MaterializeApplication(
```

Comment:
Can you make it so that all instances of `MaterializeApplication` can take advantage of those command-line arguments?

Reply:
There are only 2 occurrences of MaterializeApplication in cloudtest, and both of them are taking advantage of the cmdline parameters. Do you mean outside of cloudtest too?

Reply:
Oh sorry I now see that this is already the case :

```
@pytest.fixture(scope="session")
def mz(pytestconfig: pytest.Config) -> MaterializeApplication:
    return MaterializeApplication(
        release_mode=(not pytestconfig.getoption("dev")),
        aws_region=pytestconfig.getoption("aws_region"),
        log_filter=pytestconfig.getoption("log_filter"),
    )
```

So ACK, my fault, thanks.